web/style.css

```
@@ -383,7 +385,25 @@ input[type="search"]::placeholder { font-size: .82em; }
+   格の色のまま描かれるので、どちらの格でも破線になることをここが言う
+   (panel.mjs の duoRule)。
+
+   刻みは border ではなく背景で描く。1 本が 10px しかなく、その幅では
+   border-top-style: dashed が刻みを 1 本に丸めて実線に見えるためである。
+   3px / 2px は、隣に並ぶ 20px の dashed の見本が実際に描かれる刻みに合わせて
+   ある——同じ帯の中で、破線の刻みが二種類あるように見えない。 */
+.legend .swatch.duo { display: flex; height: 3px; border-top: 0; }
+.legend .swatch.duo > span {
+  width: 10px; height: 3px;
+  background-image: repeating-linear-gradient(
+    to right, currentColor 0 3px, transparent 3px 5px
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)